index.html

```diff
@@ -0,0 +1,285 @@
+    <script type="text/javascript" src='//cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/js/toastr.min.js'></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Poiret+One|Maven+Pro|Montserrat' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Cedarville+Cursive' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/css/toastr.min.css">
+    <link rel="stylesheet" href="vendors/normalize.css">
+    <link rel="stylesheet" href="styles/ionicons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/module.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/state.css">
+  </head>
+  <body>
+  <div id='body-wrapper'></div>
+  <div id='nav-title-wrapper'>
+    <nav>
+    <div id='nav-details'>
+      <div id="mobile-menu" class='float-left'>
+        <span class="ion-navicon-round float-left"></span>
+      </div>
+      <div id='nav-title'>My Funny Friend and Me</div>
+    </div>
+    <div id='nav-selection'>
+      <ul>
+        <li class="tab"><a href="/" id="homeLink" class="ion-happy maven ivory-text">HOME</a></li>
+        <li class="tab"><a href="#" id="favoritesLink" class="ion-ios-paw maven ivory-text">FAVORITE PETS</a></li>
+        <li class="tab"><a href="#" id="aboutLink" class="ion-person maven ivory-text">ABOUT US</a></li>
+        <li class="tab"><a href="#" id="resourcesLink" class="ion-link maven ivory-text">RESOURCES</a></li>
+      </ul>
+    </div>
+    </nav>
+  </div>
+    <div id='main-wrapper'>
+    <section id='greeting-page' class='section-wrapper'>
+      <div>
+        <h1 id='greeting-text' class='maven ivory-text'>...Your new pet is just a few clicks away! Click here to start...</h1>
+      </div>
+    </section>
+    <section id="stage-1" class='section-wrapper'>
+        <div id='stage-1-wrapper'>
+          <h1 class='maven ivory-text'>I would like to adopt a...</h1>
+        <div class='choice-container'>
+          <button class="petButton" value='dog' id='dog-btn'>DOG</button>
+          <button class="petButton" value='cat' id='cat-btn'>CAT</button>
+        </div>
+        <!-- section where the random pets will go in desktop view -->
+          <h2 id='randomPetMessage' class='maven ivory-text'>Click me for some pets that might interest you.</h2>
+          <div id='randomResultsWrapper'>
+
+          </div>
+      </div>
+    </section>
+
+
+    <section id="searchSection" class='section-wrapper'>
+      <div id="searchSectionWrapper">
+        <h1 class='maven ivory-text'>and I live in...</h1>
+        <form id="zipFindForm">
+          <input id="zipFind" type="text" name="zip" placeholder="zipcode" required>
```

Comment:
Hitting the `enter` key to submit this refreshes the page. That default action needs to be accounted for.